doc/general_documentation/INSTALL_melodic.md

```diff
@@ -1,3 +1,5 @@
+\page install-melodic Install in Ubuntu 18.04 / ROS melodic
+
```

Comment:
Looking at this for the first time... I think we should try to combine these into one file so it's easier to maintain. The only differences are the numbers in some scripts as well as install_luajit and install_opencv, right? Could we call these two from one of the 18_04 scripts, then have one INSTALL file with just a branch at those two scripts?

Reply:
I merged it in the same readme file but did not add the install_lua and install_opencv inside because they need sudo. This is a problem, but it breaks the dockerfile that can't have sudos in it. I'll look better into it in the future.